app/html/mod/reportdetails.html

```diff
@@ -208,6 +208,18 @@ <h3>@{_('Comment Preview:')}</h3>
               </span>
             </div>
           </div>
+
+          <hr>
+          <div class="col-12 admin-page-form">
+            <form  method="POST" class="ajaxform pure-form" action="@{url_for('do.create_report_note', type=report['type'], id=report['id'])}" data-reload="true">
+              <h3>@{_('Leave note on report')}</h3>
+              @{createreportenote.csrf_token()!!html}
+                @{createreportenote.text(placeholder="Leave a note")!!html}
```

Comment:
Placeholder should be translatable

Reply:
It might also be good to pass `required=True` to let the frontend do the field required validation